<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Augmentation</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../Desktop/사무실/워너버스이노베이션 로고.png</normaloff>../../../Desktop/사무실/워너버스이노베이션 로고.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_9">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>150</y>
      <width>171</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Transformation And Edge</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="F_CB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Flip</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="R_CB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Rotation</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="PT_CB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Perspective Transform</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="E_CB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Edge</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_10">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>171</width>
      <height>261</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Noise And Blur</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="GN_CB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Gaussian Noise</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="PN_CB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Poisson Noise</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="CDO_CB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Coarse Drop Out</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="CCO_CB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Coarse Channel Out</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="SN_CB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Speckle Noise</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="SAP_CB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Salt And Pepper Noise</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="BB_CB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Basic Blur</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="MB_CB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Median Bulr</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>171</width>
      <height>261</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="Bright_CB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Bright</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="OCA_CB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>One Color Add</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="TCA_CB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Two Color Add</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="T3CA_CB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Three Color Add</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="To_Gray_CB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Image To Gray</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="TCR_CB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Two Channel Remove</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="CS_CB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel Shuffle</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="CLAHE_CB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>CLAHE</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="CR_CB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel Remove</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="HE_CB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Histogram Equalization</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>531</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QComboBox" name="GT_Type_COMB">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>76</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Yolo txt</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TF csv</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pascal Xml</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="Image_Directory_LE">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>361</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>../Input/</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Image_Directory_B">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Input Path</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_38">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>GT Type</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Output_Directory_LE">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>361</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>../Output/</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Output_Directory_B">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Output Path</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="Run_B">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>410</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Augmentation Run</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>351</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>548</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
